docs/components/modules/ROOT/pages/bean-language.adoc

```diff
@@ -20,7 +20,7 @@ methods.
 The xref:manual::bean-binding.adoc[Bean Binding] rules are used to bind the
 xref:manual::message.adoc[Message] Exchange to the method parameters; so you can
 annotate the bean to extract headers or other expressions such as
-xref:components::xpath-language.adoc[XPath] or xref:components::xquery-language.adoc[XQuery] from the message.
+xref:xpath-language.adoc[XPath] or xref:xquery-language.adoc[XQuery] from the message.
```

Comment:
These languages come from components so best not change this.

Reply:
I think, when I was fixing those errors they were there, but when I try to push my changes to the camel It showed me conflictions, I accept some of the incomming changes and some of the current changes which I thought the best approach. Then after I saw that Mr.Zoran has pushed fixes just 1  hour before.
May I revert this commit?

Reply:
Yup, please do.

Reply:
Hello Mr.Zoran, can you check now?